### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by Megha-Joshi**
In this PR, I have added responsiveness in the SHADOW app.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The code is neatly structured
Good to merge

File: src/pages/Homepage/homepage.jsx [hunk 1]
```diff
@@ -33,50 +32,54 @@ removeItemFromWatchLaterVideos(video._id, token) : addItemToWatchLaterVideos(tok
...
-    content: {
-      width: "18rem",
-      height: "20rem",
-      margin: "5rem auto",
-      backgroundColor: "var(--accent-color)",
-    },
-  };
+overlay: {
+top: "6rem",
+backgroundColor: "rgba(52, 58, 64, 0.8)",
```
Comment (rutu14) [inline src/pages/Homepage/homepage.jsx:57]: Can add variable for the color

File: src/video-components/Sidebar/sidebar.css [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
+.nav-list {
     padding: 10px;
     text-align: start;
     cursor: pointer;
+    width: max-content;
 }
 
-.nav-list:hover{
+.nav-list:hover {
     background-color: #909090;
```
Comment (rutu14) [inline src/video-components/Sidebar/sidebar.css:21]: Can add a variable for color

**Review by rutu14 — APPROVED** [review]
The code is good to merge.

File: src/pages/Homepage/homepage.jsx (hunk 1 above)
Comment (Megha-Joshi) [reply src/pages/Homepage/homepage.jsx:57]: It is only used here.

File: src/video-components/Sidebar/sidebar.css (hunk 2 above)
Comment (Megha-Joshi) [reply src/video-components/Sidebar/sidebar.css:21]: Okay

**Pull request merged**